stratos/chain.json

```diff
@@ -88,11 +88,11 @@
       {
         "address": "https://rpc-stratos.whispernode.com:443",
         "provider": "WhisperNode 🤐"
+      },
       {
         "address": "https://stratos-rpc.noders.services:443",
         "provider": "[NODERS]TEAM"
       }
```

Comment:
@JeremyParish69 I'm a little stumped, this change looks correct but when I view the file it's still showing me the old commit.